src/Infrastructure/Services/UserService.cs

```diff
@@ -399,6 +404,13 @@ public async Task<Response<string>> RespondToMfaChallengeAsync(string mfaCode, s
             try
             {
                 var mfaLoginDetails = DeserializeMfaLoginDetails(mfaDetails);
+                if (_devSettings.CurrentValue.BypassMfa)
```

Comment:
Do we set ASPNETCORE_ENVIRONMENT in our environment deployments? Perhaps we should.. and then add that as an extra layer of protection here, as this could easily be misconfigured in PROD.

Reply:
currently being set to Production for all envs, I agree that this would be a sensible addition but may require a bigger re-write due to secrets and when they are configured based off env but I will take a look Monday